grass7/imagery/i.modis/i.modis.download/i.modis.download.py

```diff
@@ -195,6 +199,8 @@ def main():
                           "the username and password"))
     # set username, password and folder by file
     else:
+        if os.path.isdir(options['settings']):
```

Comment:
```suggestion
        if not os.path.isfile(options['settings']):
```

Reply:
I committed the code but I don't see it in the code....